src/ts/plugins/grapesjs-theme-mode.ts

```diff
@@ -0,0 +1,105 @@
+//TEST 10
+import grapesjs, { Editor, PluginOptions, CssRule } from 'grapesjs'
+
+export default (editor: Editor, opts: PluginOptions = {}) => {
+  //console.log('Plugin grapesjs-theme-mode chargé !');
+
+  //define the current mode to both
+  (editor as any).ThemeMode = { current: 'both' }
+
+  //add the sector to the StyleManager
+  const injectThemeSector = () => {
+    const sm = editor.StyleManager
+    if (sm.getSectors().some((sec) => sec.getId() === 'theme-mode')) { //to avoid the duplication
+      return
+    }
+
+    sm.addSector('theme-mode', {
```

Comment:
Don't add a themeMode selector in the Style Manager. It doesn't belong there.
You want your selector to be somewhere in the top bar, like the responsive / device / breakpoint selector

Here:
![image](https://github.com/user-attachments/assets/a97382c4-bff5-4b18-b1ef-54dab57d08d3)

Or maybe ask the community about the UX

But for sure this doesn't belong to the "css styles" part